README.md

```diff
@@ -2,93 +2,132 @@
 
 <img alt="ORBIT Logo" src="logos/orbit_logo_simple.png" align="right" width="520" >
 
-The **O**pen **R**untime **B**inary **I**nstrumentation **T**ool, is a standalone **C/C++** profiler for Windows and Linux. Its main purpose is to help developers visualize the execution flow of a complex application.
-
-The key differentiator with many existing tools is that no alteration to the target process is necessary. Orbit does not require you to change a single line of code. It doesn't require you to recompile or even relaunch the application you want to profile. Everything is done seamlessly, right when you need it. It requires zero integration time and zero iteration time.
-
-Orbit combines sampling and dynamic instrumentation to optimize the profiling workflow. Sampling can quickly identify interesting functions to instrument. Dynamic instrumentation results in exact function entry and exit information which is presented in the form of per-thread hierarchical call graphs.  Scheduling events are also shown to visualize when a thread was running and on what core.
+Orbit, the **O**pen **R**untime **B**inary **I**nstrumentation **T**ool is a
+standalone **native** application (C, C++, Rust, Go, ...) profiler for Windows
```

Comment:
I think the parentheses make this sentence hard to read. 

Maybe something like: 
" ... standalone native application profiler for Windows and Linux. It supports native applications written in languages such as C, C++, Rust, or Go."